File: src/main/java/fit/wenchao/db/JavaVarName.java
package fit.wenchao.db;

import lombok.Data;
import lombok.var;

@Data
public class JavaVarName {

    String name;

    public static JavaVarName fromUnderScore(String name) {
        var javaVarName = new JavaVarName();
        String underScore = VarCaseConvertUtils.lowerUnderScore2LowerCamel(name);
        javaVarName.setName(underScore);
        return javaVarName;
    }

    private void setName(String name) {
        this.name = name;
    }
}

File: src/main/kotlin/fit/wenchao/db/JavaVarName.kt
package fit.wenchao.db;


class JavaVarName {
    var name :String ? = null;
}

fun  fromUnderScore(name:String ): JavaVarName {
    var javaVarName = JavaVarName()
    var underScore = VarCaseConvertUtils.lowerUnderScore2LowerCamel(name);
    javaVarName.name = underScore;
    return javaVarName;
}
